package.json

```diff
@@ -101,6 +101,7 @@
     "buffer": "5.4.3",
     "form-data": "2.5.1",
     "localforage": "1.7.3",
+    "moment": "2.24.0",
```

Comment:
`moment` is something we probably won't end up using since it's such a big dependency, so please remove this and reset `package-lock.json`

Reply:
I added `moment` to our dependencies because it's something we are actually using at MonthOverview, PaymentDetails and PaymentsList components. And all direct dependencies must be linked on `package.json`

If we really want to stop using it (removing it), that's an issue for another PR, not this one. :)

Reply:
It was working fine before this PR and it will work fine after you remove it here... please remove. We want to get rid of as many dependencies as humanely possible.

Reply:
That other issue exists in the form of #372. However I've created a new issue specifically for moment (#874). Please still remove it here tho.

Reply:
> It was working fine before this PR and it will work fine after you remove it here

It works fine because we have Cypress. If tomorrow Cypress doesn't need `moment` anymore, GI installation will break. Btw, there is an existing JS rule on eslint and node community called "no-extraneous-dependencies". See [ref 1](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md) and [ref 2](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-missing-import.md)

But anyway... removed.